Packages/com.unity.inputsystem/InputSystem/Devices/Touchscreen.cs

```diff
@@ -559,7 +560,6 @@ protected override void FinishSetup()
             base.FinishSetup();
 
             primaryTouch = GetChildControl<TouchControl>("primaryTouch");
-            displayIndex = primaryTouch.displayIndex;
```

Comment:
Why is this line removed ?

I'd suggest Hakan review this PR since he added this code:
https://github.com/Unity-Technologies/InputSystem/pull/1623

Reply:
It's not needed any more since I added the other attributes which override display index and use the state from the primary touch.
Like other overridden values like pressure.